mozilla_django_oidc/auth.py

```diff
@@ -75,7 +75,14 @@ def filter_users_by_claims(self, claims):
 
     def verify_claims(self, claims):
         """Verify the provided claims to decide if authentication should be allowed."""
-        return 'email' in claims
+        scopes = import_from_settings('OIDC_RP_SCOPES', 'openid email')
+        if 'email' in scopes.split():
```

Comment:
Maybe add a short comment explaining that this line is here to facilitate the default configuration.